examples/basic-jsx-external/index.html

```diff
@@ -10,12 +10,16 @@
     <h1>Basic Example with External JSX</h1>
     <div id="container">
       <p>
-        If you can see this, React is not working right. This is probably because you&apos;re viewing
-        this on your file system instead of a web server. Try running
+        If you can see this, React is not working right. This is probably because you&apos;re viewing this on your file system instead of a web server.<br>
+        Try navigating to the React root 
+        <pre>
+          cd /path/to/react
+        </pre>
+        and running
         <pre>
           python -m SimpleHTTPServer
         </pre>
-        and going to <a href="http://localhost:8000/">http://localhost:8000/</a>.
+        and going to <a href="http://localhost:8000/examples/basic-jsx-external/">http://localhost:8000/examples/basic-jsx-external/</a>.
```

Comment:
Can we revert this whole file? We have the same thing in the quadratic example, both working under the assumption that you run `python ...` from the directory where the file is. We'll overhaul the example sometime soon so these are better and more consistent, but let's not make them more inconsistent in the meantime.